integrationtest/src/it/jsonOutput/verify.groovy

```diff
@@ -0,0 +1,5 @@
+String jsonReport = new File(basedir, "target/validationReport.json").text
+
+assert jsonReport.contains("\"totalViolations\" : 5")
+assert jsonReport.contains("\"totalIgnoredViolations\" : 0")
+assert jsonReport.contains("\"groupedBy\" : \"rule\"")
```

Comment:
I guess it's difficult to change into JSON assertions? (e.g. using Java test or other groovy functionality) 
String comparison for JSON can be a bit brittle; but OK if too much work to change.